tests/integration/conftest.py

```diff
@@ -119,7 +119,7 @@ def split_sample_data_dir_state_edit(tmpdir_factory, split_sample_data_dir):
 
 
 @pytest.fixture(scope="module")
-def user_config():
+def overrides():
```

Comment:
Nit: this seems like it should be called `config`?

Reply:
Ah, good catch